src/rlgl.h

```diff
@@ -519,6 +519,7 @@ RLAPI unsigned int rlLoadAttribBuffer(unsigned int vaoId, int shaderLoc, void *b
 RLAPI void rlglInit(int width, int height);           // Initialize rlgl (buffers, shaders, textures, states)
 RLAPI void rlglClose(void);                           // De-inititialize rlgl (buffers, shaders, textures)
 RLAPI void rlglDraw(void);                            // Update and draw default internal buffers
+RLAPI void rlglCheckErrors(void);                     // Check and log OpenGL error codes
```

Comment:
About naming, I'd prefer to call it just `rlCheckErrors()`, actually `rlglDraw()` is going to be removed and replaced by `rlDrawRenderBatch()`, added lately, so the library will just keep `rlglInit()` and `rlglClose()` that looks more consistent to me.

Reply:
Sounds good. Done.